sdks/python/apache_beam/transforms/external_test.py

```diff
@@ -66,6 +68,32 @@
 # pylint: enable=wrong-import-order, wrong-import-position
 
 
+def retry_on_grpc_timeout(max_retries=5, delay=10, max_total_time=300):
```

Comment:
I don't love the idea of forcing test retries since this will be equally true for customer jobs. Can we try the fix from @liferoad instead (https://github.com/apache/beam/pull/36528)?